server/tests/test_notebook_view.py

```diff
@@ -151,6 +152,43 @@ def test_tryit_view(client, fake_user, logged_in, iomd):
         assert len(response.redirect_chain) == 0
 
 
+@pytest.mark.parametrize("file_size", [1, 10485764])
```

Comment:
There's no need to create an actual 10 megabyte file here, you can use the settings fixture to change `MAX_FILE_SIZE` for this test. There's an existing example of this here:

https://github.com/iodide-project/iodide/blob/cfc9722816ec5d2de70bc43ced56fcdc19b3d639/server/tests/test_file_update_operations.py#L56